src/test/run-make/issue-88756-default-output/Makefile

```diff
@@ -0,0 +1,4 @@
+-include ../../run-make-fulldeps/tools.mk
+
+all:
+	$(RUSTDOC) 2>&1 | diff - output-default.stdout
```

Comment:
This is not testing the right thing. The original issue was that -W help didn't show the lint help.

Reply:
OK. Shall I add test for default behavior and `-W help` as both seem to have been mentioned?